scene/3d/audio_stream_player_3d.h

```diff
@@ -117,7 +117,7 @@ class AudioStreamPlayer3D : public Node3D {
 	float _get_attenuation_db(float p_distance) const;
 
 	float panning_strength = 1.0f;
-	float cached_global_panning_strength = 1.0f;
+	float cached_global_panning_strength = 0.5f;
```

Comment:
Does not seem like this is set anywhere?

Reply:
It's set here: https://github.com/godotengine/godot/blob/f2a61684143af02a8cbe7002645af817607f9bd6/scene/3d/audio_stream_player_3d.cpp#L916

Likewise in 2D: https://github.com/godotengine/godot/blob/f2a61684143af02a8cbe7002645af817607f9bd6/scene/2d/audio_stream_player_2d.cpp#L475